.scalafmt.conf

```diff
@@ -0,0 +1,33 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+version = 3.7.1
+runner.dialect = scala212
+maxColumn = 96
+docstrings.style = keep
+indent.defnSite = 2
+indent.extendSite = 2
```

Comment:
I notice that `indent.caseSite` (not overridden here yet) has 4 as its default size.  That parameter applies only to a long case expression separated by | which spans multiple lines and decides how much to indent the additional lines past the "case" on the first line.  Interestingly enough, the parameter's example shows a formatted case expression with `indent.caseSite = 5` which aligns all the parts of the expression.  It seems like the default value should've been 2 rather than 4 for consistency, but I'm not sure what actual Daffodil code is using.

The easiest way to find out would be to do a new commit with only a new `indent.caseSite = 2` line in this file and see what new changes appear in Daffodil files as a result.  If current Daffodil code is already indented 4 spaces (we would have to check the same files from the main branch before any scalafmt formatting was done), then we'll reject the change and keep using scalafmt's default value.